### PR TITLE
Add config and templateDir tags

### PR DESCRIPTION
**Pull request by thucngyyen**
Add support for templateDir and config tag:
- Both can be passed as single file
- TemplateDir works with passing in directory

Add CI workflow to make sure generator works, however, missing the functionality to check on generated artifact. Will create following issue for this.

Note for @wing328 : you might want to enable protection rule on master

### CODE REVIEW

**Comment by thucngyyen** [comment]
@wing328  can you take a look?

**Comment by wing328** [comment]
@thucngyyen thanks for the PR 👍 

Can you please PM me via Slack when you've time to have a quick chat (IM) on this?

https://join.slack.com/t/openapi-generator/shared_invite/zt-12jxxd7p2-XUeQM~4pzsU9x~eGLQqX2g

File: BUILD.bazel [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Copyright 2019 OpenAPI-Generator-Bazel Contributors
 
-load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
+load("@rules_pkg//:pkg.bzl", "pkg_tar", "pkg_deb")
```
Comment (thucngyyen) [inline BUILD.bazel:3]: @wing328 pkg_tar is deprecated (extracted) from bazel, have to import seperately: https://docs.bazel.build/versions/2.2.0/be/pkg.html

How it imported: https://github.com/OpenAPITools/openapi-generator-bazel/pull/41/files#diff-5493ff8e9397811510e780de47c57abb70137f1afe85d1519130dc3679d60ce5

**Comment by wing328** [comment]
Tests passed via https://github.com/OpenAPITools/openapi-generator-bazel/pull/42 

Thanks for the PR and the CI tests 👍

File: BUILD.bazel [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Copyright 2019 OpenAPI-Generator-Bazel Contributors
 
-load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
```
Comment (chloegonzales) [inline BUILD.bazel:3]: Tristengonzales002@outlook.com

Comment (chloegonzales) [inline BUILD.bazel:3]: Genevasalinas53@yahoo.com

**Pull request merged**